1.9/usage/gpu-support.md

```diff
@@ -0,0 +1,224 @@
+    echo "Public SubnetId: $public_subnet"
+    ```
+
+1. Run the script from from your local terminal with a single argument of stack prefix (STACK_NAME).
+    ```bash
+    $ bash ./zen.sh dcos
+    ```
+    
+    The output should look like this:
+    
+    ```
+    Creating Zen Template Dependencies
+    VpcId: vpc-e0bd2c84
+    InternetGatewayID: igw-38071a5d
+    Private SubnetId: subnet-b32c82c5
+    Public SubnetId: subent-b02c55c4
+    ```
+    
+1. Use these dependency values as input to create your stack in CloudFormation in the next steps.
+
+#### Deploy the CF template
+
+1. Go to [CloudFormation](https://console.aws.amazon.com/cloudformation/home) and click **Create Stack**.
+
+1. On the **Select Template** page, specify the custom template that you created on your master node via the `sudo bash dcos_generate_config.sh --aws-cloudformation` command . For example, `
+https://s3-us-west-2.amazonaws.com/joel-test-gpu/templates/dcos/config_id/fe4f3d47e23eda85301c875a28a34a8f48e44f95/cloudformation/el7-zen-1.json`.
+
+1. On the **Specify Details** page, specify these values and and click **Next**.
+    - **StackName** - Specify your cluster name.
+    - **AdminLocation** - Accept default.
+    - **CustomAMI** - Specify the custom AMI for your region:
+
+      us-west-2: ami-9b5d97fb
+      us-east-1: ami-e10e50f6
+      ap-southeast-2: ami-37b28f54
+
+    - **InternetGateway** - Specify the InternetGatewayId value from your zen template dependencies (e.g. igw-f15f0895).
+    - **KeyName** - Specify your AWS EC2 key pair.
+    - **MasterInstanceType** - Accept the default master instance type (e.g. `m3.xlarge`).
+    - **OAuthEnabled** - Indicate whether to enable OAuth.
+    - **PrivateAgentInstanceCount** - Specify the number of private agent nodes.
+    - **PrivateAgentInstanceType** - Specify a machine type of `g2.2xlarge`. This is the GPU-supported machine type.
+    - **PrivateSubnet** - Specify the Private SubnetId value from your zen template dependencies (e.g. subnet-e7193fbf).
+    - **PublicAgentInstanceCount** - Specify the number of public agent nodes.
+    - **PublicAgentInstanceType** - Specify a machine type of `g2.2xlarge`. This is the GPU-supported machine type.
+    - **PublicSubnet** - Specify the Public SubnetId value from your zen template dependencies (e.g. subnet-98193fc0).
+    - **Vpc** - Specify the VpcId value from your zen template dependencies (e.g. vpc-49a0202e).
+
+1. On the Options page, accept the defaults and click Next.
+    **Tip**: You can choose whether to rollback on failure. By default this option is set to Yes.
+
+1. On the Review page, check the acknowledgement box, then click **Create**.
+    **Tip**: If the **Create New Stack** page is shown, either AWS is still processing your request or you’re looking at a different region. Navigate to the correct region and refresh the page to see your stack.
+
+## Configure a Non-AWS Cluster
```

Comment:
Can we maybe put the Non-AWS instructions first and maybe stress how trivial it actually is to enable GPU support (just a single added configuration parameter and the installation of a library dependency).

All of the complication we see in the instructions comes from the AWS specific confiration / installation that needs to be done, and I think it distracts from what is really going on.  It's necessary to have those instructions here (rather than simply linking to them), but I think the way things are organized currently distracts from what is really necessary just to get GPU support working.